packages/interfaces/src/store.ts

```diff
@@ -15,7 +15,7 @@ export interface Cursor {
   digest: Uint8Array;
   receiverTime: bigint;
   senderTime: bigint;
-  pubsubTopic: string;
+  pubSubTopic: string;
```

Comment:
This is  link to the protobuf definition so I would not change it without changing the protobuf